schema/deploy/database_functions/prune_orphan_swrs_orgs_and_facilities.sql

```diff
@@ -0,0 +1,46 @@
+-- Deploy ggircs-portal:database_functions/prune_orphan_swrs_orgs_and_facilities to pg
+-- requires: schema_ggircs_portal_private
+
+-- This function looks at all the ciip portal organisations and facilities, and removes them if their swrs corresponding entry is not found.
+-- Items which don't have a swrs id were added in CIIP and should be kept.
```

Comment:
use a `comment on function` so that this is visible in the DB as well

Reply:
Excellent point 👌